include/openssl/aes.h

```diff
@@ -68,14 +68,27 @@ void AES_ofb128_encrypt(const unsigned char *in, unsigned char *out,
                         size_t length, const AES_KEY *key,
                         unsigned char *ivec, int *num);
 /* NB: the IV is _two_ blocks long */
-void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
-                     size_t length, const AES_KEY *key,
-                     unsigned char *ivec, const int enc);
+DEPRECATEDIN_1_2_0(void AES_ige_encrypt(const unsigned char *in,
+                                        unsigned char *out, size_t length,
+                                        const AES_KEY *key, unsigned char *ivec,
+                                        const int enc))
 /* NB: the IV is _four_ blocks long */
-void AES_bi_ige_encrypt(const unsigned char *in, unsigned char *out,
-                        size_t length, const AES_KEY *key,
-                        const AES_KEY *key2, const unsigned char *ivec,
-                        const int enc);
+DEPRECATEDIN_1_2_0(void AES_bi_ige_encrypt(const unsigned char *in,
```

Comment:
Well we don't actually have a DEPRECATEDIN_1_1_1 macro at the moment. I was under the impression that we could not deprecate in a binary compat release (or am I wrong?). On the other hand scheduling a future deprecation is not without precedent: https://github.com/openssl/openssl/blob/master/include/openssl/asn1t.h#L918.

In other words although we can't deprecate now, we can still *plan* to deprecate in the future - so this PR is still targetting 1.1.0